### DISCUSSION TITLE
[Feature] Ability to turn off Auto Darken

### DISCUSSION THREAD

**Discussion by rrrhys** [immich-app/immich#12248] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The auto-darken when backups are running is quite hard to view if I am watching the backup run. I still have a huge backlog to upload and it gets stuck regularly, so it's good to see at a glance.

Is there a config flag or toggle somewhere to disable this feature?

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
The auto darken feature no longer exists, so this request is obsolete.
